### PR TITLE
Framework: Upgrade ESLint computed-property-spacing to error

### PR DESCRIPTION
**Pull request by Copons**
Related: #8562

This pull request continues the crusade to upgrade all ESLint rules to errors by resolving remaining issues with [`computed-property-spacing` rule](http://eslint.org/docs/rules/computed-property-spacing).

**Testing Instructions:**

That CircleCI passes is indication that the changes included are working as intended.

Verify that no regressions have been introduced to affected sections.

### CODE REVIEW

File: client/lib/connections-list/list.js [hunk 1]
```diff
@@ -0,0 +1,591 @@
+/**
```
Comment (aduth) [inline client/lib/connections-list/list.js]: Why was this file added? Guessing erroneous rebase.

**Comment by Copons** [comment]
@aduth rebased and removed that file that got included somehow!
